index.html

```diff
@@ -242,32 +221,46 @@ <h2><dfn>measureWithOptions()</dfn> method</h2>
           <li>
             Compute <var>start time</var> as follows:
             <ol>
-              <li>If <var>measureOptions</var>'s <a>startTime</a> member is present, let <var>start time</var> be the value returned by running the <a>convert a mark to a timestamp</a> algorithm passing in <var>measureOptions</var>'s <a>startTime</a>.</li>
+              <li>If <var>startOrMeasureOptions</var> is present and is a <a>PerformanceMeasureOptions</a> object, and if its <a>start</a> member is present, let <var>start time</var> be the value returned by running the <a>convert a mark to a timestamp</a> algorithm passing in <var>startOrMeasureOptions</var>'s <a>start</a>.</li>
               <li>
-                Otherwise, if <var>measureOptions</var>'s <a>duration</a> and <a>endTime</a> members are both present:
+                Otherwise, if <var>startOrMeasureOptions</var> is present and is a <a>PerformanceMeasureOptions</a>, and if its <a>duration</a> and <a>end</a> members are both present:

```

Comment:
"is a PerformanceMeasureOptions object" to be consistent with the rest

Reply:
Done